roofit/roofitcore/inc/RooSpan.h

```diff
@@ -82,12 +82,15 @@ class RooSpan {
   _span{beginIn, sizeIn}
   { }
 
-
+  /// Construct from a std::vector.
+  template<typename U = T,
+      typename = typename std::enable_if<std::is_const<U>::value, void>::type>
```

Comment:
why is it not enough to have just the `const&` version?

Reply:
I cannot reproduce the problem that I thought this was causing. It was probably something else which I had fixed in the mean time.
I removed the commit, and will go back to this if a problem comes up again.